tests/Http/Place/UpdateAddressRequestHandlerTest.php

```diff
@@ -0,0 +1,255 @@
+            $this->commandBus->getRecordedCommands()
+        );
+    }
+
+    /**
+     * @test
+     */
+    public function it_supports_multiple_languages(): void
+    {
+        $updateAddressRequest = $this->psr7RequestBuilder
+            ->withRouteParameter('placeId', self::PLACE_ID)
+            ->withRouteParameter('language', 'nl')
+            ->withBodyFromString(
+                '{
+                    "streetAddress": "Veldstraat 11",
+                    "postalCode": "9000",
+                    "addressLocality": "Gent",
+                    "addressCountry": "BE"
+                }'
+            )
+            ->build('PUT');
+
+        $updateAddressRequest2 = $this->psr7RequestBuilder
+            ->withRouteParameter('placeId', self::PLACE_ID)
+            ->withRouteParameter('language', 'fr')
+            ->withBodyFromString(
+                '{
+                    "streetAddress": "Rue du champ 11",
+                    "postalCode": "9000",
+                    "addressLocality": "Gand",
+                    "addressCountry": "BE"
+                }'
+            )
+            ->build('PUT');
+
+        $this->updateAddressRequestHandler->handle($updateAddressRequest);
+        $this->updateAddressRequestHandler->handle($updateAddressRequest2);
+
+        $this->assertEquals(
+            [
+                new UpdateAddress(
+                    self::PLACE_ID,
+                    new Address(
+                        new Street('Veldstraat 11'),
+                        new PostalCode('9000'),
+                        new Locality('Gent'),
+                        new CountryCode('BE')
+                    ),
+                    new Language('nl')
+                ),
+                new UpdateAddress(
+                    self::PLACE_ID,
+                    new Address(
+                        new Street('Rue du champ 11'),
+                        new PostalCode('9000'),
+                        new Locality('Gand'),
+                        new CountryCode('BE')
+                    ),
+                    new Language('fr')
+                ),
+            ],
+            $this->commandBus->getRecordedCommands()
+        );
+    }
+
+    /**
+     * @test
+     */
+    public function it_throws_on_invalid_country(): void
+    {
+        $updateAddressRequest = $this->psr7RequestBuilder
+            ->withRouteParameter('placeId', self::PLACE_ID)
+            ->withRouteParameter('language', 'nl')
+            ->withBodyFromString(
+                '{
+                    "streetAddress": "Veldstraat 11",
+                    "postalCode": "9000",
+                    "addressLocality": "Gent",
+                    "addressCountry": "BEL"
+                }'
+            )
+            ->build('PUT');
+
+        $this->expectException(InvalidArgumentException::class);
+
+        $this->updateAddressRequestHandler->handle($updateAddressRequest);
+    }
+
+    public function it_throws_on_empty_values(): void
```

Comment:
Very good to add these checks already, but because the name of the method is plural `values` it's maybe better to also check this for the other properties?